pages/common/dict.md

```diff
@@ -0,0 +1,24 @@
+# dict
+
+> Client/server software, human language dictionary databases, and tools supporting the DICT protocol.
+> More information: <https://github.com/cheusov/dictd>.
+
+- List available databases:
+
+`dict -D`
+
+- Get database information:
+
+`dict -i {{database_name}}`
```

Comment:
Do you just need the name or also the path?

Reply:
Just the name, which comes from using `dict -D`